composer.json

```diff
@@ -28,7 +28,7 @@
     "require": {
         "php": "^7.2",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-        "slevomat/coding-standard": "dev-master#9401aac3b076250dfcbfe391a53f9fea4b2c5201",
+        "slevomat/coding-standard": "dev-master#469368cc6d8fe83aceba259ef65f1c0a2f63055b",
```

Comment:
Why does this hash change again?

Reply:
Slevomat meanwhile fixed this https://github.com/doctrine/coding-standard/pull/137#discussion_r350748628

Reply:
ok I thought this was unintentional